app/src/main/res/layout/fragment_edit_board.xml

```diff
@@ -18,14 +18,36 @@
         app:contentInsetStartWithNavigation="0dp"
         app:titleMarginStart="0dp">
 
-        <TextView
-            android:id="@+id/board_title"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:text="placeholder"
-            android:textColor="#F5F5F5"
-            android:textSize="22sp"
-            android:textStyle="bold" />
+        <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
You don't need this wrapper since all you define in this view is width and height. I don't think it's doing anything. Let's flatten the view hierarchy as much as possible for performance.

But also just to note - nested ConstraintLayouts in particular are very expensive and generally should be avoided